lib/plugins/backup.rb

```diff
@@ -138,7 +145,29 @@ def render(placeholder)
       end
 
       def log_message
-        "Uploading '#{@target_name}' to '#{@bucket}'"
+        "Uploading '#{@target_name}' to '#{@target_bucket}'"
+      end
+
+      def iam_account
+        iam_client.list_account_aliases.account_aliases.first
```

Comment:
What happens in an account without aliases? I wonder what we want the behavior to be.